### DISCUSSION TITLE
is there any difference between the free and paid versions?

### DISCUSSION THREAD

**Discussion by doobre24** [yermak/AudioBookConverter#578] [Q&A]
I just started using the app and wonder if the paid version allows access to the settings/parameters/config etc..?
I would like to change the exported file naming, correct the field mapping between Mp3tag and ABC and other simple things.
cheers for this smooth running app, that is way better than any others that I have tried.. CD

**Comment by yermak** [comment]
There is only one difference - free version offer you to get new version from Stream everytime you run it, the rest is the same. Frankly speaking, this is unintentional bug - but I decided not to fix it :)
As for other advanced settings, I might add them in future (they used to be accessible via config file, but I changed it now), they should be available in interface at some point.

**Comment by doobre24** [comment]
Hi there, thanks for your prompt reply.I look forward to any future updates.Cheers CD 


Sent from Yahoo Mail for iPhone


On Tuesday, March 19, 2024, 06:56, Yarick Yermak ***@***.***> wrote:




There is only one difference - free version offer you to get new version from Stream everytime you run it, the rest is the same. Frankly speaking, this is unintentional bug - but I decided not to fix it :)
As for other advanced settings, I might add them in future (they used to be accessible via config file, but I changed it now), they should be available in interface at some point.

**Comment by petenun** [comment]
I have purchased and am running the Steam version, but have opted to run the free version instead. The prompt to upgrade is far less painful than waiting for Steam to load first and then dismiss the subsequent annoying splash screens. Personally, I would rather see a Donate option on the free version instead of dealing with Steam.
